Fixed null values being displayed in Tasks Tab of User Info Screen.

ayushs136 opened the pull request:
<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->

**What kind of change does this PR introduce?**
Fixed null values being displayed in Tasks Tab of User Info Screen. The screen is now getting data from database, which user has filled. 

**Issue Number:**
#409 

**Did you add tests for your code?**
No

**Snapshots/Videos:**
![image](https://user-images.githubusercontent.com/30320414/111782944-a73fe880-88df-11eb-84c1-359177c0e5be.png)

**Does this PR introduce a breaking change?**
Yeah

**Other information:**
Also Formatted date in dd-mm-yyyy format.

**Have you read the contributing guide?**
Yeah

Sagar2366 commented:
1. @ayushs136 seems like this issue is not yet fixed properly.
For example, check for test Test user users task list, it is throwing error.

2. Also, you have mentioned in PR description that this PR introduces braking change, can you please explain more on that ?

3. Please share video demo after adding changes for this fix.

<img width="426" alt="Screenshot 1942-12-29 at 12 09 14 PM" src="https://user-images.githubusercontent.com/26805882/111861717-6646e300-8976-11eb-9c4f-741ec45d1a2d.png">

ayushs136 commented:
@Sagar2366 Made the required changes. Please review.

Sagar2366 commented:
@ayushs136 please add unit test for your code.

Sagar2366 commented:
@ayushs136 can you please pull latest changes added to master for travis ci fix and send PR  again for review ?
Also disable code formatter and remove changes which are not required to fix the issue.

ayushs136 commented on lib/views/pages/organization/switch_org_page.dart:
```diff
@@ -126,70 +120,73 @@ class _SwitchOrganizationState extends State<SwitchOrganization> {
-          return Divider();
-        },
-      ),
+              padding: EdgeInsets.only(top: 10.0),
```
Okay

ayushs136 replied:
@Sagar2366 there was another issue in switch_org_page.dart, and it contains that changes, I missed to make a issue for that but you can find the screenshots attached to it here.

From
![image](https://user-images.githubusercontent.com/30320414/112114200-c7242480-8bdd-11eb-9e83-c171bfda785a.png)

To
![image](https://user-images.githubusercontent.com/30320414/112114279-e327c600-8bdd-11eb-80ba-1e6c0e5e6584.png)

ayushs136 commented:
Fixed all issues and reverted unused changes. Please check out @Sagar2366! Meanwhile can I create another issue for the above switch_org_page.dart one?
